### ISSUE TITLE
Deploy command does not work if using serverless 2.30 or later

### ISSUE THREAD

**Issue by huksley** [prisma-labs/serverless-plugin-typescript#238]
Since this release
https://github.com/serverless/serverless/releases/tag/v2.30.0

This plugin does not work, if running with SLS_DEBUG="*" 
it produces following:

```
....
Serverless: Invoke deploy
Serverless: Warning: The command you entered did not catch on any hooks
```
